content/en/docs/ops/integrations/kyverno-authz-server/index.md

```diff
@@ -0,0 +1,464 @@
+---
+title: Kyverno Authz Server
+description: How to integrate with Kyverno Authz Server.
+weight: 33
+keywords: [integration,kyverno,policy,authorization]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+The [Kyverno Authz Server](https://kyverno.github.io/kyverno-envoy-plugin) is a GRPC authorization server implementing the [Envoy's External Authorization protocol](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/auth/v3/external_auth.proto).
+
+Istio and the Kyverno Authz Server is a solid option to deliver policy quickly and transparently to application team everywhere in the business, while also providing the data the security teams need for audit and compliance.
+
+## Try it out
```

Comment:
Probably best to follow the pattern set out by docs like https://istio.io/latest/docs/tasks/security/authorization/authz-custom/